utilities/project_helpers/scripts/update-message-xml-translations.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+import lxml.etree as lxml
```

Comment:
I'll be a bit nitpicky here, but do we need `lxml`?

Reply:
Aha, probably not. I'll try without.